src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -92,20 +92,185 @@
       <cd-card cardTitle="Used Capacity"
                i18n-title
                class="col-sm-2 d-flex w-100 h-50 pb-3"
-               aria-label="Details card">
+               aria-label="Used Capacity">
         <span  class="ms-4 me-4 text-center">
           <h1>{{ totalPoolUsedBytes | dimlessBinary}}</h1>
         </span>
       </cd-card>
       <cd-card cardTitle="Avg Object Size"
                i18n-title
                class="col-sm-2 d-flex w-100 h-50 pt-3"
-               aria-label="Details card">
+               aria-label="Avg Object Size">
         <span class="ms-4 me-4 text-center">
           <h1>{{ averageObjectSize | dimlessBinary}}</h1>
         </span>
       </cd-card>
     </div>
   </div>
+  <div class="row pt-4">
+    <cd-card cardTitle="Multisite Sync Status"
+             i18n-title>
+      <div *ngIf="!showMultisiteCard"
+           class="text-center">
+           No Data!
+      </div>
```

Comment:
showing `No Data` while the data is loading is misleading. We should actually show a skeleton of some kind but since the bootstrap skeletons are not available in 5.0 which we are using, we can just try to work with what we got. We have a `LoadingComponent` which we use in the forms when the form is loading. Probably that could fit here!

We should also differentiate between `loading` and `No Data`. When its loading it should be `loading` and when the load is done and then still no data, then `No Data`.

 and btw `No Data` doesn't make any sense, it should be an info alert panel which says something like `Multisite needs to be configured inorder to see the multisite sync status`.

Reply:
also you should use templates for scenarios like this and `*ngIf; else`.

`ngIf="showMultisiteCard; else notConfigured`
```
<ng-template #notConfigured>
   Multisite needs to be configured inorder to see the multisite sync status
</ng-template>

```